### PR TITLE
UI redesign of the menu, location chooser and list pages.

### PR DESCRIPTION
**Pull request by awalkowiak**

### CODE REVIEW

**Comment by awalkowiak** [comment]
@jmiranda, @mdpearson this one is ready

File: grails-app/conf/Config.groovy [hunk 1]
```diff
@@ -331,7 +331,7 @@ jqueryValidationUi {
 
 
 // Allow users to customize logo image url as well as label
-openboxes.logo.url = "https://openboxes.com/img/logo_30.png"
+openboxes.logo.url = "/openboxes/images/logo/openboxes_logo_40x40.jpg"
```
Comment (jmiranda) [inline grails-app/conf/Config.groovy:334]: /openboxes should probably be replaced with ${appName}

Comment (jmiranda) [reply grails-app/conf/Config.groovy:334]: With that said, we can push that change off until afterwards.

File: grails-app/controllers/org/pih/warehouse/api/ProductApiController.groovy [hunk 2]
```diff
@@ -24,6 +27,40 @@ class ProductApiController extends BaseDomainApiController {
     def grailsApplication
     def productAvailabilityService
 
+    def list = {
```
Comment (jmiranda) [inline grails-app/controllers/org/pih/warehouse/api/ProductApiController.groovy:30]: We need to remember to include some warnings about this breaking API change.

**Review by jmiranda — APPROVED** [review]
LGTM. I got a quarter of the way through. I can't do any more. We'll fix the broken in the release branch.

**Pull request merged**